src/gui/screen_menu_sensor_info_MK3.5.cpp

```diff
@@ -19,6 +19,12 @@
 
 void ScreenMenuSensorInfo::windowEvent(EventLock /*has private ctor*/, window_t *sender, GUI_event_t event, void *param) {
     if (event == GUI_event_t::LOOP) {
+
+#if HAS_TEMP_BOARD
```

Comment:
This file is specific to MK3.5. The ifdef is not necessary here.

Reply:
Has been changed. Thanks.